files/src/index.ts

```diff
@@ -1,5 +1,7 @@
 import App from './main';
 import { ComponentManager, setPropertyDidChange } from '@glimmer/component';
+import initializeCustomElements from '@glimmer/web-component';
+import customElementDefinitions from './config/custom-element-definitions';
```

Comment:
Can we eliminate the config file and just make it an array of component names you pass?